code/game/machinery/ARES/ARES_interface_admin.dm

```diff
@@ -0,0 +1,491 @@
+	data["nuketimelock"] = NUCLEAR_TIME_LOCK
+	data["nuke_available"] = datacore.nuke_available
+
+	var/list/logged_announcements = list()
+	for(var/datum/ares_record/announcement/broadcast as anything in datacore.records_announcement)
+		var/list/current_broadcast = list()
+		current_broadcast["time"] = broadcast.time
+		current_broadcast["title"] = broadcast.title
+		current_broadcast["details"] = broadcast.details
+		current_broadcast["ref"] = "\ref[broadcast]"
+		logged_announcements += list(current_broadcast)
+	data["records_announcement"] = logged_announcements
+
+	var/list/logged_alerts = list()
+	for(var/datum/ares_record/security/security_alert as anything in datacore.records_security)
+		var/list/current_alert = list()
+		current_alert["time"] = security_alert.time
+		current_alert["title"] = security_alert.title
+		current_alert["details"] = security_alert.details
+		current_alert["ref"] = "\ref[security_alert]"
+		logged_alerts += list(current_alert)
+	data["records_security"] = logged_alerts
+
+	var/list/logged_flights = list()
+	for(var/datum/ares_record/flight/flight_log as anything in datacore.records_flight)
+		var/list/current_flight = list()
+		current_flight["time"] = flight_log.time
+		current_flight["title"] = flight_log.title
+		current_flight["details"] = flight_log.details
+		current_flight["user"] = flight_log.user
+		current_flight["ref"] = "\ref[flight_log]"
+		logged_flights += list(current_flight)
+	data["records_flight"] = logged_flights
+
+	var/list/logged_bioscans = list()
+	for(var/datum/ares_record/bioscan/scan as anything in datacore.records_bioscan)
+		var/list/current_scan = list()
+		current_scan["time"] = scan.time
+		current_scan["title"] = scan.title
+		current_scan["details"] = scan.details
+		current_scan["ref"] = "\ref[scan]"
+		logged_bioscans += list(current_scan)
+	data["records_bioscan"] = logged_bioscans
+
+	var/list/logged_bombs = list()
+	for(var/datum/ares_record/bombardment/bomb as anything in datacore.records_bombardment)
+		var/list/current_bomb = list()
+		current_bomb["time"] = bomb.time
+		current_bomb["title"] = bomb.title
+		current_bomb["details"] = bomb.details
+		current_bomb["user"] = bomb.user
+		current_bomb["ref"] = "\ref[bomb]"
+		logged_bombs += list(current_bomb)
+	data["records_bombardment"] = logged_bombs
+
+	var/list/logged_deletes = list()
+	for(var/datum/ares_record/deletion/deleted as anything in datacore.records_deletion)
+		if(!istype(deleted))
+			continue
+		var/list/current_delete = list()
+		current_delete["time"] = deleted.time
+		current_delete["title"] = deleted.title
+		current_delete["details"] = deleted.details
+		current_delete["user"] = deleted.user
+		current_delete["ref"] = "\ref[deleted]"
+		logged_deletes += list(current_delete)
+	data["records_deletion"] = logged_deletes
+
+	var/list/logged_discussions = list()
+	for(var/datum/ares_record/deleted_talk/deleted_convo as anything in datacore.records_deletion)
+		if(!istype(deleted_convo))
+			continue
```

Comment:
remove any instance of this pattern, using `as anything` and then typechecking it anyway is just pointless. remove the as anything and the istype